src/v/net/server.cc

```diff
@@ -89,6 +89,12 @@ void server::start() {
             if (cfg.listen_backlog.has_value()) {
                 lo.listen_backlog = cfg.listen_backlog.value();
             }
+            if (cfg.tcp_recv_buf.has_value()) {
```

Comment:
Isn't this equivalent to just direct assignment, like:

```
lo.so_rcvbuf = cfg.tcp_recv_buf;
```

Reply:
I guess yeah. Changed.